applications/ws-chem-ner/package-app.json

```
@@ -0,0 +1,34 @@
+{
+	"private": true,
+	"name": "ws-chem-ner",
+	"dependencies": {
+		"@ezs/core": "3.1.0",
```

```suggestion
		"@ezs/core": "3.2.2",
```

Il faut vérifier que de nouvelles versions ne sont pas sorties. Ici, c'est le cas. Et ça corrige des problèmes de gestion d'erreur.